src/components/Form/Form.js

```diff
@@ -1,32 +1,34 @@
 import React, { useState } from 'react';
 import './Form.css';
 
-const Form = ({searchMakeup}) => {
-    const [userInput, setUserInput] = useState('');
-const updateForm = (event) => {
-    setUserInput(event.target.value.toLowerCase())
-    }
+const Form = ({ searchMakeup, handleClick}) => {
+  const [userInput, setUserInput] = useState("");
+  const updateForm = (event) => {
+    setUserInput(event.target.value.toLowerCase());
+  };
 
-    const filterSearchMakeup = (event) => {
-        event.preventDefault();
-        searchMakeup(userInput);
-        setUserInput('');
-    }
-    return (
-        <form className='searchForm'>
-            <input
-                type='text'
-                placeholder='Search by Brand or Tag (e.g.: "natural")'
-                value={userInput}
-                onChange={updateForm}
-                aria-label='Search Makeup Items'
-                aria-required='true'
-            >
-            </input>
-            <button onClick={filterSearchMakeup} className='searchButton'>Search</button>
-        </form>
-    )
-}
+  const filterSearchMakeup = (event) => {
+    event.preventDefault();
+    searchMakeup(userInput);
+    setUserInput("");
+    // handleClick()
```

Comment:
I don't think we need this anymore, right? That means we can also delete that variable that we were trying to pass down earlier.